tests/handlers/test_koji.py

```diff
@@ -48,6 +48,50 @@ def test_build_complete(mock_koji_cs, mock_getBuild_two):
     build.commit.is_connected(commit)
 
 
+@mock.patch('koji.ClientSession')
+def test_modulebuild_complete(mock_koji_cs, mock_getBuild_module_complete,
+                              modulebuild_getTag, mock_getBuild_complete):
+    """Test the Koji handler when it recieves a new build complete message."""
+    mock_koji_session = mock.Mock()
+    mock_koji_session.getBuild.return_value = mock_getBuild_module_complete
+    mock_koji_session.getTag.return_value = modulebuild_getTag
+    mock_koji_session.listTaggedRPMS.return_value = [[], [mock_getBuild_complete]]
+    mock_koji_cs.return_value = mock_koji_session
+
+    with open(path.join(message_dir, 'koji', 'modulebuild_complete.json'), 'r') as f:
+        msg = json.load(f)
+    assert KojiHandler.can_handle(msg) is True
+    handler = KojiHandler(config)
+    handler.handle(msg)
+
+    build = ModuleKojiBuild.nodes.get_or_none(id_='753795')
+    # Regular Koji Build attributes
+    assert build is not None
+    assert build.completion_time == datetime(2018, 8, 17, 16, 54, 17, tzinfo=pytz.utc)
+    assert build.creation_time == datetime(2018, 8, 17, 16, 54, 29, tzinfo=pytz.utc)
+    assert build.epoch is None
+    assert build.extra == ('{"typeinfo": {"module": {"modulemd_str": "module", "name": "virt",'
+                           ' "stream": "rhel", "module_build_service_id": 1648, '
+                           '"version": "20180817161005", "context": "9edba152", '
+                           '"content_koji_tag": "module-virt-rhel-20180817161005-9edba152"}}}')
+    assert build.id_ == '753795'
+    assert build.name == 'virt'
+    assert build.release == '20180817161005.9edba152'
+    assert build.version == 'rhel'
+    assert build.start_time == datetime(2018, 8, 17, 16, 10, 29, tzinfo=pytz.utc)
+    assert build.state == 1
+    # Additional Module Koji Build attributes
+    assert build.context == '9edba152'
+    assert build.mbs_id == 1648
+    assert build.module_name == 'virt'
+    assert build.module_stream == 'rhel'
+    assert build.module_version == '20180817161005'
+
+    commit = DistGitCommit.nodes.get_or_none(hash_='cf0bd75c564366cc03b882a94cdcb9da0945690a')
+
+    build.commit.is_connected(commit)
```

Comment:
Could you also verify that the module build has a component attached to it please?